README.md

```diff
@@ -149,21 +149,18 @@ Build from source:
 Install the client
 -------------------
 
-We recommend that you version pin the confluent-kafka-go import to v1:
-
 Manual install:
 ```bash
-go get -u gopkg.in/confluentinc/confluent-kafka-go.v1/kafka
+go get -u github.com/confluentinc/confluent-kafka-go/kafka
```

Comment:
For people on older versions of Go where go mods are not supported we'll still want them to use the versioned import, right?

So I think we'll want to maintain the current instructions but also add a section on using modules.